cl/opinion_page/views.py

```diff
@@ -796,7 +797,15 @@ async def view_opinion_old(
     unbound form.
     """
     # Look up the court, cluster, title and note information
-    cluster: OpinionCluster = await aget_object_or_404(OpinionCluster, pk=pk)
+    cluster: OpinionCluster = await aget_object_or_404(
+        OpinionCluster.objects.prefetch_related(
+            Prefetch(
+                "sub_opinions",
+                queryset=Opinion.objects.order_by("ordering_key"),
```

Comment:
About the case when there are opinions with both NULL and valid `ordering_key` values, it will not show the "Combined" or scraped opinion first
![image](https://github.com/user-attachments/assets/1351997f-1e26-4c68-83a5-074f1b122752)

However, if you change
```python
from django.db.models import F
...
queryset=Opinion.objects.order_by(F("ordering_key").asc(nulls_first=True))
```

it will use the desired order
![image](https://github.com/user-attachments/assets/a459e18e-aedb-4668-9b65-bcfd74a52ad4)

Reply:
thanks @grossir